tests/roots/test-ext-javadoctest-maven/index.rst

```diff
@@ -0,0 +1,27 @@
+Test Java With Third Dependencies Handled by Maven
+==================================================
+
+In case we need to test documentation for projects that consume Java native libraries and
+Third party Java dependencies, then, is needed to configure/add these dependencies thru a
```

Comment:
- In case -> if
- Third party -> third-party (it is used as an adjective) 
- thru -> through (thru is informal)
- then, is needed -> then these dependencies need to be configured or added through a [..]
- dependency management -> dependency management tool 
- suchs as -> such as Maven or Gradle. 
- in this case innitially -> Currently, only Maven is supported. 

This file should not be inside the testroot but should be part of the Sphinx documentation itself. For tests, only the example is needed (but the example must also be included in the documentation).